.github/workflows/linux-build-base.yml

```
@@ -24,45 +24,72 @@ on:
-
-      - name: Build Artifact
+      - name: Build Velox
         env:
-          VELOX_DEPENDENCY_SOURCE: BUNDLED
-          Boost_SOURCE: SYSTEM
-          ICU_SOURCE: SYSTEM
-          MAKEFLAGS: MAX_HIGH_MEM_JOBS=4 MAX_LINK_JOBS=3
+          # Tune parallelism based on runner type: self-hosted (16-core) vs GitHub-hosted (4-core)
+          # MAX_HIGH_MEM_JOBS: High-memory compilation tasks (templates, large TUs)
+          # MAX_LINK_JOBS: Parallel linking tasks (memory-intensive)
+          MAKEFLAGS: >-
```

actionlint (1.7.9)

property "labels" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)